test/cpp/api/misc.cpp

```diff
@@ -153,3 +155,179 @@ TEST_CASE("make_unique") {
     REQUIRE(ptr[2] == 0);
   }
 }
+
+TEST_CASE("ordered-dict") {
```

Comment:
I am going to keep shouting about these catch tests until they get ported to gtest. PORT IT TO GTEST. PORT IT TO GTEST.

Reply:
I know this has to happen and I'll be excited to use gtest, but I don't want to spend time on something like this right now. H1 deliverables have precedence